release-calendar/src/types.ts

```diff
@@ -22,26 +22,52 @@ export enum Channel {
   OPT_IN_BETA = 'opt-in-beta',
   OPT_IN_EXPERIMENTAL = 'opt-in-experimental',
   NIGHTLY = 'nightly',
+  ROLLBACK = 'rollback',
+  CREATED = 'created',
 }
 
-//TODO: switch from class to interface once data is coming from server
-//for now, it must be a class because of data.ts
-// export interface Release {
-//   name: string;
-//   channel: Channel;
-//   date: Date;
-//   isRollback: boolean;
-// }
+export interface Release {
+  name: string;
+  promotions: Promotion[];
+  cherrypicked?: Release;
+}
 
-export class Release {
+export class Release implements Release {
```

Comment:
Apparently creating a class also creates its type, so this is redundant - https://www.typescriptlang.org/docs/handbook/classes.html#using-a-class-as-an-interface

I think it's valid because interfaces can be combined - https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces

🤷🏻‍♀️I'm gonna just remove it 🤷🏻‍♀️